RuntimePermissions/app/src/main/java/sample/github/nisrulz/runtimepermissions/MainActivity.java

```diff
@@ -62,7 +59,7 @@ public void onClick(DialogInterface dialogInterface, int i) {
                     }).setPositiveButton("Allow", new DialogInterface.OnClickListener() {
                 @Override
                 public void onClick(DialogInterface dialogInterface, int i) {
-                    ActivityCompat.requestPermissions(this, new String[]{Manifest.permission.CALL_PHONE}, 1);
+                    ActivityCompat.requestPermissions(MainActivity.this, new String[]{Manifest.permission.CALL_PHONE}, 1);
                 }
```

Comment:
Thanks though. Btw your repository is just awesome for every level of Android Developer.